code/modules/projectiles/gun.dm

```diff
@@ -261,7 +261,15 @@
 			return
 
 	if(flag)
-		if(user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		var/simplified_mode = user.client?.prefs.read_player_preference(/datum/preference/choiced/zone_select) == PREFERENCE_BODYZONE_SIMPLIFIED
+		var/mob/living/living_target = target
+		if (!simplified_mode)
+			if(user.is_zone_selected(BODY_ZONE_PRECISE_MOUTH))
+				handle_suicide(user, target, params)
+				return
+		// On simplified mode, contextually determine if we want to suicide them
+		// If the target is ourselves, they are buckled, restrained or lying down then suicide them
```

Comment:
this is really goofy ngl dont likey.

Reply:
The only times I have used this function is when a target has been incapacitated or is buckled/arrested. We have to determine it contextually because we want to still allow people to do this action but we don't want it to accidentally trigger during combat.

Reply:
This is basically guesswork. I don't think I have ever used this action and now I could see myself accidentally using it on someone because i clicked them while on the ground with my gun on head mode.

Reply:
I'll change it from lying to being incapacitated since you might want to shoot someone lying down but someone who is fully incapacitated at point blank range probably would be more likely to want this. There are no changes to the non-precise system